Description: Remove flto linker flag for static lib
 <long description that can span multiple lines, optional>
Author: <name and email of author, optional>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gatb-core/src/CMakeLists.txt
+++ b/gatb-core/src/CMakeLists.txt
@@ -83,11 +83,16 @@
 
 include_directories (${gatb-core-includes})
 
+# Compile static lib first
+# Temporary remove the -flto linker flag
+string(REPLACE "-flto" "" TEMP_CXX_FLAGS ${CMAKE_CXX_FLAGS})
 add_library   (gatbcore-static  STATIC  ${ComponentFiles} )
 target_link_libraries (gatbcore-static  hdf5 z pthread)
 set_target_properties (gatbcore-static   PROPERTIES OUTPUT_NAME  gatbcore  clean_direct_output 1)
 set_target_properties (gatbcore-static   PROPERTIES VERSION "${gatb-core-version}"
-                                                    SOVERSION "${gatb-core_VERSION_MAJOR}")
+                                                    SOVERSION "${gatb-core_VERSION_MAJOR}"
+                                                    COMPILE_FLAGS "${TEMP_CXX_FLAGS}")
+# Compile dynamic lib
 add_library   (gatbcore-dynamic SHARED  ${ComponentFiles} )
 target_link_libraries (gatbcore-dynamic hdf5 z pthread)
 set_target_properties (gatbcore-dynamic  PROPERTIES OUTPUT_NAME  gatbcore  clean_direct_output 1)
